Update misleading documentation about importing compressed WAV files (#10729)
godotengine/godot-docs: tutorials/assets_pipeline/importing_audio_samples.rst

Each format has different advantages:

> - WAV files use raw data or light compression (IMA-ADPCM or QOA). Currently, they can only be imported in raw format, but Godot can apply compression to already imported files. They are 

Author: ```suggestion
- WAV files use raw data or light compression (IMA-ADPCM or QOA). Currently, they can only be imported in raw format, but Godot allows compression after import. They are 
```

Reviewer: That's perfect!

> - WAV files use raw data or light compression (IMA-ADPCM or QOA). Currently, they can only be imported in raw format, but Godot allows compression after import. They are 
>   lightweight to play back on the CPU (hundreds of simultaneous voices in this
>   format are fine). The downside is that they take up a lot of disk space.

Reviewer: ```suggestion
- WAV files use raw data or light compression (IMA-ADPCM or QOA). Currently
  they can only be imported in raw format, but Godot allows compression after
  import. They are lightweight to play back on the CPU (hundreds of simultaneous
  voices in this format are fine). The downside is that they take up a lot of disk space.
```
